### PR TITLE
Embed an informational built-by string in JIT

### PR DESCRIPTION
**Pull request by jakobbotsch**
cc @dotnet/jit-contrib

### CODE REVIEW

**Review by kunalspathak — APPROVED** [review]
LGTM

**Comment by jkotas** [comment]
What is going to consume this string?

**Comment by jakobbotsch** [comment]
> What is going to consume this string?

Sorry, I should have put that in the description. It's item 3 here: https://github.com/dotnet/runtime/pull/68292#issuecomment-1105200762

**Comment by jkotas** [comment]
It may be useful to include it as a code comment.

File: src/coreclr/jit/buildstring.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#define STRINGIFY(L) #L
```
Comment (jkotas) [inline src/coreclr/jit/buildstring.cpp:1]: Nit: Add license header

**Pull request merged**